vstools/utils/misc.py

```diff
@@ -73,5 +104,6 @@ def padder(
 @disallow_variable_format
 @disallow_variable_resolution
 def pick_func_stype(clip: vs.VideoNode, func_int: FINT, func_float: FFLOAT) -> FINT | FFLOAT:
+    """Pick the sample type for a function."""
```

Comment:
pick the function from the sample type of the clip's format

Reply:
Ooooh, so kinda like that dumb function I had. I get it now.